pytest/tests/sanity/memtrie_disktrie_switch.py

```diff
@@ -135,13 +135,20 @@ def __next_target_height(self, num_epochs):
         stop_height = random.randint(1, EPOCH_LENGTH)
         return current_height + num_epochs * EPOCH_LENGTH + stop_height
 
-    def next_nonce(self, signer_key):
+    def __next_nonce(self, signer_key):
         """Returns the next nonce to use for sending transactions for the given signing key."""
         assert signer_key in self.nonces
         nonce = self.nonces[signer_key]
         self.nonces[signer_key] = nonce + 42
         return nonce
 
+    def __retrieve_nonces(self, account_keys):
+        """Retrieves the next nonce for the accounts and stores them locally."""
+        for account_key in account_keys:
+            nonce = self.rpc_node.get_nonce_for_pk(account_key.account_id,
+                                                   account_key.pk) + 42
```

Comment:
Why do we need to add 42?

Reply:
adding 1 should work, I just wanted to add a more arbitrary number. checked with +1 only and it worked. done.